### PR TITLE
Ashwini

### PR DESCRIPTION
**Pull request by AshwiniDhaneppagol**
Hi everyone file is ready for review. Please have a look

### CODE REVIEW

File: src/Room3/Ashwini01.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package Room3;
+
+public class Ashwini01 {
+    public static void main(String[] args) {
+        System.out.println("Hi...");
```
Comment (yavuzvelioglu) [inline src/Room3/Ashwini01.java]: Hi @AshwiniDhaneppagol you typed Hi so i accept

**Review by yavuzvelioglu — APPROVED** [review]

File: src/Room3/Ashwini01.java (hunk 1 above)
Comment (MonikaSturman) [inline src/Room3/Ashwini01.java]: Hi Ashwini please print Hello Word!

File: src/Ashwini.java [hunk 2]
```diff
@@ -0,0 +1,3 @@
+public class Ashwini {
+
+}
```
Comment (meral-das) [inline src/Ashwini.java:3]: please  do not forget to add "Hello World!"

**Comment by MonikaSturman** [comment]
Thank you very much for your work!

File: src/Ashwini.java (hunk 2 above)
Comment (AshwiniDhaneppagol) [reply src/Ashwini.java:3]: Done! Please check

**Comment by meral-das** [comment]
great job!

File: src/Room3/Ashwini01.java (hunk 1 above)
Comment (AshwiniDhaneppagol) [reply src/Room3/Ashwini01.java]: Done !Please check

**Review by MonikaSturman — COMMENTED** [review]
Thank you!!

**Review by MonikaSturman — APPROVED** [review]

**Pull request merged**